sample_data/README.md

```diff
@@ -29,6 +29,26 @@ To use the sample data, at minimum 8 GB memory should be allocated to the contai
 
 ### Use Notebook image
 
+1. Use `docker start 2020_full_ch_server` to start the container.
+
+2. Go to the src folder in the open-digger root directory, create a file named 'local_config.ts' with the following contents:
+
+   ```python
+   export default {
+       db: {
+           clickhouse: {
+                host: '172.17.0.1'
+           }
+       }
+   }
+   ```
+
+3. Use `npm run build` to build the project.
```

Comment:
`npm run notebook` contains `npm run build`, you don't need to add this step before `notebook` command.